docs/0.5/references/cli/splinterd.1.md

```diff
@@ -1,4 +1,9 @@
 % SPLINTERD(1) Cargill, Incorporated | Splinter Commands
+<!--
```

Comment:
The man pages should only be updated once they are updated in splinter

Reply:
They've been updated in https://github.com/Cargill/splinter/pull/904 and https://github.com/Cargill/splinter/pull/905